host/3.0/buster/amd64/dotnet/dotnet-inproc/dotnet-composite.template

```diff
@@ -10,7 +10,8 @@ ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
     HOME=/home \
     FUNCTIONS_WORKER_RUNTIME=dotnet \
     DOTNET_USE_POLLING_FILE_WATCHER=true \
-    HOST_VERSION=${HOST_VERSION}
+    HOST_VERSION=${HOST_VERSION} \
```

Comment:
I might have missed this in the emails, but why do we need HOST_VERSION?

Reply:
can you expand? this is an existing setting

Reply:
sorry, i was using the wrong diff view :)